addon/chart-data-updater.js

```diff
@@ -22,6 +22,10 @@ export default Ember.Object.extend({
       if (chart.datasets[0].bars.length !== datasets[0].data.length) {
         return this.set('redraw', true);
       }
+    } else if (typeof chart.datasets[0].points !== 'undefined') {
+      if (chart.datasets[0].points.length !== datasets[0].data.length) {
```

Comment:
You will actually want to iterate through `datasets` since you could have multiple lines and one of them could update length.  I had addressed this in a PR back in September but the library doesn't seem to be maintained:
https://github.com/aomran/ember-cli-chart/pull/37/files